Sources/SwiftDiagnostics/DiagnosticsFormatter.swift

```diff
@@ -73,12 +73,15 @@ public struct DiagnosticsFormatter {
   /// Number of lines which should be printed before and after the diagnostic message
   public let contextSize: Int
 
-  /// Whether to colorize formatted diagnostics.
-  public let colorize: Bool
```

Comment:
The removal of the `colorize` property is an API-breaking change. 

To maintain source compatibility, could you add

```swift
@available(*, deprecated, message: "Store the `colorize` property passed to the initializer instead")
public var  colorize: Bool {
  return diagnosticDecorator is ANSIDiagnosticDecorator
}

Reply:
Done